InterfaceF/src/Server/DataBass.java

```diff
@@ -0,0 +1,104 @@
+package Server;
+import java.sql.*;
+import java.util.concurrent.ExecutionException;
+
+public class DataBass  {
+    public static Connection conn;
+    public static Statement statement;
+    public static ResultSet resultSet;
+public static void Run()  throws SQLException, ClassNotFoundException{
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.